src/Thread/Post.js

```diff
@@ -113,6 +117,13 @@ const Post = ({
                 onClick={() => updateQuote("", postKey)}
                 styles={theme.ReplyButton}
               />
+              <UpvoteButton
+                user={user}
+                upvotes={Object.keys(post.likes || {}).length || 0}
+                liked = {Object.keys(post.likes || {}).includes(uid || {}) && post.likes[uid] === true}
```

Comment:
This could use [lodash's includes()](https://lodash.com/docs/3.10.1#includes) instead of native ES6 .includes() for IE 11 support.